### DISCUSSION TITLE
Failed to execute the linker

### DISCUSSION THREAD

**Discussion by byrialsen** [unoplatform/Uno.Wasm.Bootstrap#511] [Q&A]
Hi 

I followed this guide:
[Hosting Uno Platform WebAssembly apps on Azure Static Web Apps](https://platform.uno/blog/hosting-uno-platform-webassembly-apps-on-azure-static-web-apps/)

But I have some issues releated to the Github Action part. It fails with an ```Failed to execute the linker``` error:

![image](https://user-images.githubusercontent.com/1816703/171813757-d0430221-e243-4f61-99a1-a7c6969b8a01.png)

Check out the repo here:

[NaboFight repo](https://github.com/byrialsen/NaboFight)

Check the failed action

[Failed action](https://github.com/byrialsen/NaboFight/runs/6718030878?check_suite_focus=true)

Does anyone know what the problem is?

**Comment by jeromelaban** [comment]
The best way to troubleshoot this is to [generate a binlog](https://platform.uno/docs/articles/uno-builds-troubleshooting.html#generating-msbuild-binary-log-files). The linker errors are specified there.

**Reply by ZeProgFactory** [reply]
Is there any progress on this issue? We have the same problem:
```
MetriShop.Client.Uno46XF.Wasm -> D:\GitWare\TopSales\MetriShop\MetriShop.Client\MetriShop.Client.Uno46XF\MetriShop.Cl
  ient.Uno46XF.Wasm\bin\Release\net5.0\MetriShop.Client.Uno46XF.Wasm.dll
C:\Users\zepro\.nuget\packages\uno.wasm.bootstrap\7.0.0-dev.243\build\Uno.Wasm.Bootstrap.targets(205,3): error : System
.Exception: Failed to execute the linker [D:\GitWare\TopSales\MetriShop\MetriShop.Client\MetriShop.Client.Uno46XF\Metri
Shop.Client.Uno46XF.Wasm\MetriShop.Client.Uno46XF.Wasm.csproj]
C:\Users\zepro\.nuget\packages\uno.wasm.bootstrap\7.0.0-dev.243\build\Uno.Wasm.Bootstrap.targets(205,3): error :    at
Uno.Wasm.Bootstrap.ShellTask_v067084231ea16a1fe0aab02299f18104309dc4b0.RunPackager() in D:\a\1\s\src\Uno.Wasm.Bootstrap
\ShellTask.cs:line 972 [D:\GitWare\TopSales\MetriShop\MetriShop.Client\MetriShop.Client.Uno46XF\MetriShop.Client.Uno46X
F.Wasm\MetriShop.Client.Uno46XF.Wasm.csproj]
C:\Users\zepro\.nuget\packages\uno.wasm.bootstrap\7.0.0-dev.243\build\Uno.Wasm.Bootstrap.targets(205,3): error :    at
Uno.Wasm.Bootstrap.ShellTask_v067084231ea16a1fe0aab02299f18104309dc4b0.Execute() in D:\a\1\s\src\Uno.Wasm.Bootstrap\She
llTask.cs:line 254 [D:\GitWare\TopSales\MetriShop\MetriShop.Client\MetriShop.Client.Uno46XF\MetriShop.Client.Uno46XF.Wa
sm\MetriShop.Client.Uno46XF.Wasm.csproj]
```

Here is the compressed binlog:
[msbuild.zip](https://github.com/unoplatform/Uno.Wasm.Bootstrap/files/9960235/msbuild.zip)

**Reply by jeromelaban** [reply]
@ZeProgFactory your error is this one:
```
[00:00:25.7713037] ILLink: error IL1009: Assembly reference 'System.Windows.Extensions' could not be resolved.
```

You probably have some code that uses this particular dependency, and it's likely that it won't work under .NET WebAssembly.

**Reply by ZeProgFactory** [reply]
Oups ... I didn't find any reference to 'System.Windows.Extensions' in our source code. 
It's only mentioned as dependency in some webservices.
